bigquery_etl/shredder/config.py

```diff
@@ -176,6 +176,9 @@ def fields(self) -> tuple[str, ...]:
     client_id_target(
         table="search_derived.mobile_search_clients_daily_v1"
     ): DESKTOP_SRC,
+    client_id_target(
+        table="search_derived.mobile_search_clients_daily_v2"
+    ): DESKTOP_SRC,
```

Comment:
This should be using mobile sources instead of desktop.  I have an example in this draft https://github.com/mozilla/bigquery-etl/pull/5914/files.

I can fix this after this gets merged if that works for you